gpc-api-facade/src/main/java/uk/nhs/adaptors/pss/gpc/service/AcknowledgeRecordService.java

```diff
@@ -26,7 +26,7 @@ public class AcknowledgeRecordService {
     public Boolean handleAcknowledgeRecord(
             @NotNull @NotEmpty String conversationId,
             @NotNull @NotEmpty String confirmationResponseString) {
-        var patientMigrationRequest = patientMigrationRequestDao.getMigrationRequest(conversationId);
+        var patientMigrationRequest = patientMigrationRequestDao.getMigrationRequest(conversationId.toLowerCase());
```

Comment:
Hmm, strange as I'm pretty sure we're [uppercasing the conversation ID](https://github.com/NHSDigital/nia-patient-switching-standard-adaptor/blob/main/gpc-api-facade/src/main/java/uk/nhs/adaptors/pss/gpc/service/PatientTransferService.java#L39-L40) when it's inserted.

Reply:
You're correct, my bad!